lib/src/Base/Func/EvaluationImplementation.cxx

```diff
@@ -353,26 +358,6 @@ Point EvaluationImplementation::operator() (const Point & inP) const
   throw NotYetImplementedException(HERE) << "In EvaluationImplementation::operator() (const Point & inP) const";
 }
 
-Point EvaluationImplementation::operator() (const Point & inP,
```

Comment:
@dbarbier This will break otrobopt. The motivation behind the introduction of all these methods was to provide a symmetrical treatment of arguments and parameters of a function, as the distinction is somewhat arbitrary between them (and context-dependent such as in otbobopt). Do you have an idea on how to speed-up the evaluation of a function at a given point but for a sample of parameters, which is the actual need for otrobopt?

Reply:
Okay, I was not aware of that, and will think about it.  But only ParametricEvaluation redefined this method, is the gain worth the trouble?  Otherwise we can call setParameter in otropopt.

Reply:
PR sent to otrobopt to not use these functions anymore.